dspace-api/src/main/java/org/dspace/eperson/Group.java

```diff
@@ -201,7 +201,6 @@ public String getName() {
     void setName(String name) throws SQLException {
         if (!StringUtils.equals(this.name, name) && !isPermanent()) {
             this.name = name;
-            groupsChanged = true;
```

Comment:
@toniprieto : While I think your change is correct, it made me realize that this line probably was *meant* to be a call to `setMetadataModified()` (which will trigger a MODIFY_METADATA event, but *won't* recalculate the group2groupcache).  Otherwise, it looks like a group name change won't trigger any events, which means the Group may not be reindexed if necessary.

Reply:
You're right! I will add a new commit to add this call.